src/Layers/xrRender/HW.h

```diff
@@ -118,4 +118,4 @@ class CHW
 #endif // USE_OGL
 };
 
-extern ECORE_API CHW HW;
+//extern volatile ECORE_API CHW HW;
```

Comment:
Just NO! This changes renderer structure. It's wrong, even if it changes it a bit of things. `extern HW` should be defined here.